core/backend/src/test/integration/BriefcaseManager.test.ts

```diff
@@ -631,7 +631,7 @@ describe("BriefcaseManager (#integration)", () => {
     requestContext.enter();
 
     await IModelTestUtils.closeAndDeleteBriefcaseDb(requestContext, iModel);
-    assert.isTrue(numProgressCalls > 19000);
+    assert.isTrue(numProgressCalls > 20);

```

Comment:
use 10, since that's what i changed it to in [this](https://github.com/imodeljs/imodeljs/pull/1366) yet-to-be-merged PR (although i suspect it will still be a conflict).

Reply:
fixed